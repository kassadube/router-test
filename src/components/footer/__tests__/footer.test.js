/* eslint-disable react/react-in-jsx-scope */

import Footer from '../index';

test('# Footer', () => {
  const wrapper = shallow(<Footer />);
  expect(wrapper).toMatchSnapshot();
});


describe('rendering testing', () => {
  it('#footer  RENDER', () => {
    const component = (<Footer />);
    const wrapper = render(component);
    expect(wrapper).toMatchSnapshot();
  });
  it('#navbar not loged in user RENDER', () => {
    const component = (<Footer />);
    const wrapper = mount(component);
    expect(wrapper).toMatchSnapshot();
  });
});
