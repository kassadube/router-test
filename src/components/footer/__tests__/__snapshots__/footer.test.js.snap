exports[`rendering testing #footer  RENDER 1`] = `
<footer
  style="position:absolute;bottom:0px;width:100%;">
  <div
    class="row">
    <div
      class="col-xs-11 col-sm-9 col-md-7">
      this is a footeraadd
    </div>
  </div>
</footer>
`;

exports[`rendering testing #navbar not loged in user RENDER 1`] = `
<Component>
  <footer
    style={
      Object {
        "bottom": "0",
        "position": "absolute",
        "width": "100%",
      }
    }>
    <div
      className="row">
      <div
        className="col-xs-11 col-sm-9 col-md-7">
        this is a footeraadd
      </div>
    </div>
  </footer>
</Component>
`;

exports[`test # Footer 1`] = `
<footer
  style={
    Object {
      "bottom": "0",
      "position": "absolute",
      "width": "100%",
    }
  }>
  <div
    className="row">
    <div
      className="col-xs-11 col-sm-9 col-md-7">
      this is a footeraadd
    </div>
  </div>
</footer>
`;
